distribution/lib/Standard/Snowflake/0.0.0-dev/THIRD-PARTY/NOTICE

```diff
@@ -0,0 +1,7 @@
+Enso
+Copyright 2020 - 2024 New Byte Order sp. z o. o.
+
+'snowflake-jdbc', licensed under the The Apache Software License, Version 2.0, is distributed with the Snowflake.
```

Comment:
Not related to this PR directly but I'm thinking if we should add a 'display name' for our components in the license review tool, as it would look much better if it were:
```suggestion
'snowflake-jdbc', licensed under the The Apache Software License, Version 2.0, is distributed with the Standard.Snowflake library.
```